src/read_pool.rs

```diff
@@ -144,6 +150,22 @@ impl ReadPoolHandle {
             rx.map_err(ReadPoolError::from).await
         }
     }
+
+    pub fn is_busy(&self) -> bool {
+        match self {
+            ReadPoolHandle::FuturePools {
+                read_pool_normal, ..
```

Comment:
I think it's better to differentiate different pools, i.e., `is_busy(&self, priority: CommandPri)`.